master/usage/service-advertisement.md

```diff
@@ -3,33 +3,61 @@ title: Advertising Kubernetes services over BGP
 canonical_url: 'https://docs.projectcalico.org/master/usage/service-advertisement'
 ---
 
-{{site.prodname}} supports advertising Kubernetes service cluster IPs
-over BGP, just as it advertises pod IPs.  This means that, if your
-{{site.prodname}} deployment is configured to peer with BGP routers
-outside the cluster, those routers - plus any further upstream places
-that those routers propagate to - will be able to send traffic to a
-Kubernetes service cluster IP, and that traffic will be routed to one
-of the available endpoints for that service.
+## About advertising Kubernetes services over BGP
 
-This feature is enabled for each {{site.prodname}} node by [setting
-the `ADVERTISE_CLUSTER_IPS` environment
-variable]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/config-options#configuring-service-advertisement).
+{{site.prodname}} supports advertising Kubernetes service cluster IPs over BGP, just as it advertises pod IPs.
+This means that, if your {{site.prodname}} deployment is configured to peer with BGP routers
+outside the cluster, those routers - plus any further upstream places that those routers propagate to - will be able
+to send traffic to a Kubernetes service cluster IP, and that traffic will be routed to one of the
+available endpoints for that service.
 
-When this feature is enabled:
+You may want to enable this feature if you would like Kubernetes services to be accessible outside of the
+cluster without the need for a dedicated load balancer.
 
--  the cluster IP CIDR (for example, 10.96.0.0/12) is advertised from
-   every node in the cluster
+## Prerequisites
 
--  for each active service with `externalTrafficPolicy: Local`, the
-   cluster IP for that service is advertised as a /32 route from the
-   nodes that have endpoints for that service.
+**BGP peers**
+
+You must have established BGP connections to one or more routers outside of your Kubernetes cluster.
+See [configuring BGP peers][bgp-peers] for more information.
+
```

Comment:
Should we add something here about those external routers needing to have the equivalent of BIRD's `merge paths on;` config?  (i.e. configuration to program ECMP routes into their local kernel)

Reply:
Yep, definitely.